lib/registries/addon/overview/index/controller.ts

```diff
@@ -17,7 +17,6 @@ export default class Overview extends Controller {
         if (!this.registration || this.registration.isRoot) {
             return undefined;
         }
-
-        return this.registration.belongsTo('root').value() as Registration;
+        return this.registration.belongsTo('registeredFrom').id();
```

Comment:
The `registeredFrom` relationship points to the _project_ from which the registration is made. The `root` relationship points to the root _registration_ . I don't think this is getting the right info.

Reply:
Same comment as above, this has now been updated to use the root value component name, not the guid value of the registeredFrom registration. Also, this method is no longer needed and has been removed.